include/opt-sched/Scheduler/data_dep.h

```diff
@@ -389,7 +389,6 @@ class DataDepGraph : public llvm::opt_sched::OptSchedDDGWrapperBase,
                                 int nodeID, InstCount fileSchedOrder,
                                 InstCount fileSchedCycle, InstCount fileLB,
                                 InstCount fileUB, int blkNum);
-  void AddNode_(SchedInstruction *instPtr, InstCount instNum);
```

Comment:
Why some of these methods are being removed like AddNode_() and SetNum()?

Reply:
They are not used anywhere. I removed them because there's no point in having them if they aren't moved. They also reduce the ability to modify some of the variables (having a getter and a setter is barely any better than an exposed variable).

If we want them in the future, it's in the git history; we can resurrect the code.